src/installing-workflow/experimental-native-ingress.md

```diff
@@ -0,0 +1,80 @@
+
+
+ 
+Helm will install a variety of Kubernetes resources in the `deis` namespace.
+Wait for the pods that Helm launched to be ready. Monitor their status by running:
+
+```
+$ kubectl --namespace=deis get pods
+```
+
+You should also notice that a Kubernetes ingress has been installed on your cluster. You can view it by running:
+
+```
+$ kubectl get ingress --namespace deis
+```
+
+
+Depending on the order in which the Workflow components initialize, some pods may restart. This is common during the
+installation: if a component's dependencies are not yet available, that component will exit and Kubernetes will
+automatically restart it.
+
+Here, it can be seen that the controller, builder and registry all took a few loops before they were able to start:
+
+```
+$ kubectl --namespace=deis get pods
+NAME                          READY     STATUS    RESTARTS   AGE
+deis-builder-hy3xv            1/1       Running   5          5m
+deis-controller-g3cu8         1/1       Running   5          5m
+deis-database-rad1o           1/1       Running   0          5m
+deis-logger-fluentd-1v8uk     1/1       Running   0          5m
+deis-logger-fluentd-esm60     1/1       Running   0          5m
+deis-logger-sm8b3             1/1       Running   0          5m
+deis-minio-4ww3t              1/1       Running   0          5m
+deis-registry-asozo           1/1       Running   1          5m
+deis-workflow-manager-68nu6   1/1       Running   0          5m
+```
+
+## Install a Kubernetes Ingress Controller
+
+Now that Workflow has been deployed with the `global.exerpimental_native_ingress` flag set to `true`, we will need a Kubernetes ingress controller in place to begin routing traffic.
+
+Here is an example of how to use [traefik](https://traefik.io/) as an ingress controller for Workflow. Of course, you are welcome to use any controller you wish.
+
+```
+$ helm install stable/traefik --name deis-ingress-001 --namespace kube-system
+```
+
+## Configure DNS
+
+The experimental ingress feature requires a user to set up a hostname, and assumes the `deis.$host` convention.
+
+We need to point the `deis.$host` record to the public IP address of your ingress controller. You can get the public IP using the following command.
+
+```
+$ kubectl get svc deis-ingress-001 --namespace kube-system
+NAME               CLUSTER-IP      EXTERNAL-IP       PORT(S)                      AGE
+deis-ingress-001   10.23.253.220   104.154.159.184   80:30231/TCP,443:32264/TCP   19m
+```
+
+If we were using `deis.com` as a hostname we would need to create the following A DNS record.
+
+| Name              | Type          | Value           |
+| ----------------- |:-------------:| ---------------:|
+| deis.deis.com     | A             | 104.154.159.184 |
+
+
+Once all of the pods are in the `READY` state, and `deis.$host` resolves to the external IP found above Workflow is up an running!
+
+After installing Workflow, [register a user and deploy an application](../quickstart/deploy-an-app.md).
```

Comment:
Since it's experimental, should we provide a link in the docs to github issue reporting? Or encourage users to share their stories of success with different ingress controllers by submitting docs PRs as HOWTOs?

Not necessary, just thinking...